Implementacija/savetovaliste/app/Controllers/BaseController.php

```diff
@@ -19,40 +23,76 @@
  *
  * For security be sure to declare any new methods as protected or private.
  */
+class BaseController extends Controller {
+
+    /**
+     * Instance of the main Request object.
+     *
+     * @var IncomingRequest|CLIRequest
+     */
+    protected $request;
+
+    /**
+     * An array of helpers to be loaded automatically upon
+     * class instantiation. These helpers will be available
+     * to all other controllers that extend BaseController.
+     *
+     * @var array
+     */
+    protected $helpers = ['url', 'form'];
+
+    /**
+     * Constructor.
+     *
+     * @param RequestInterface  $request
+     * @param ResponseInterface $response
+     * @param LoggerInterface   $logger
+     */
+    public function initController(RequestInterface $request, ResponseInterface $response, LoggerInterface $logger) {
+        // Do Not Edit This Line
+        parent::initController($request, $response, $logger);
+
+        //--------------------------------------------------------------------
+        // Preload any models, libraries, etc, here.
+        //--------------------------------------------------------------------
+        $this->session = session();
+        if (!$this->session->get('controller')) {
+            $this->session->set('controller', 'Gost');
+        }
+    }
+
+    public function pregled_pitanja() {
+        $pitanjeModel = new PitanjeModel();
+        $pitanja = $pitanjeModel->findAll();
+        $this->prikaz('pregled_pitanja', ['pitanja' => $pitanja]);
```

Comment:
Ubuduće ne bi bilo loše da ide return pre prikazivanja. Ovde je na kraju funkcije tako da ne mora, al kada nije, onda siturno treba.
```suggestion
        return $this->prikaz('pregled_pitanja', ['pitanja' => $pitanja]);
```

Reply:
Napisala sam isto kao ti kod login